apps/aeutils/priv/extensions/get_peer_key.sh

```diff
@@ -7,20 +7,22 @@ PASS=${2:-"secret"}
 CODE="{ok, PK} = aec_keys:peer_pubkey(),
       binary_to_list(aec_base58c:encode(peer_pubkey, PK))."
 
-! relx_nodetool eval $CODE
+! PK=$(relx_nodetool eval $CODE)
 
 if [ $? -eq 1 ]; then
+    echo "${PK//\"}"
     exit 0
 fi
 
 CODE="{ok, Bin} = file:read_file(filename:join(\"$KEYSDIR\", \"peer_key.pub\")),
```

Comment:
I see this will still fail on node that was deployed though never started with this release. Hence the deploy script ignores errors https://github.com/aeternity/epoch/blob/877bfd2b4bb264e5d96a6e66f3458ac785e3286e/deployment/ansible/deploy.yml#L22-L23